<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Source Folder / URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="text_source"/>
     </item>
     <item>
      <widget class="QPushButton" name="button_from_url">
       <property name="text">
        <string>Search URL</string>
       </property>
       <property name="icon">
        <iconset theme="search"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_from_folder">
       <property name="text">
        <string>Open Folder</string>
       </property>
       <property name="icon">
        <iconset theme="folder"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Save to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="text_dest"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Browse</string>
       </property>
       <property name="icon">
        <iconset theme="folder"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Metadata</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_image"/>
       </item>
       <item>
        <widget class="QLabel" name="label_metadata"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Chapters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="chapter_table"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Convert?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_no_convert">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_convert_epub">
           <property name="text">
            <string>EPUB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_convert_pdf">
           <property name="text">
            <string>PDF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_convert_cbz">
           <property name="text">
            <string>CBZ</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_progress">
       <property name="text">
        <string>Downloading...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progress_bar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_start">
     <property name="text">
      <string>Start!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
